include/picongpu/particles/bremsstrahlung/Bremsstrahlung.tpp

```diff
@@ -243,10 +249,13 @@ unsigned int Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::n
 
 
 template<typename T_IonSpecies, typename T_ElectronSpecies, typename T_PhotonSpecies>
-template<typename Electron, typename Photon>
+template<typename Electron, typename Photon, typename T_Acc>
 DINLINE
-void Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::operator()
-    (Electron& electron, Photon& photon)
+void Bremsstrahlung<T_IonSpecies, T_ElectronSpecies, T_PhotonSpecies>::operator()(
```

Comment:
OPTIONAL:
according to the [coding guidelines](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md#17-template-class--struct-and-type-definitions) each template argument should be on a separate line